src/mono/wasm/build/WasmApp.targets

```diff
@@ -208,4 +246,38 @@
       <WasmAssembliesFinal Include="@(_WasmAssembliesInternal)" LlvmBitCodeFile="" />
     </ItemGroup>
   </Target>
+
+  <!-- Setup NPM packages for NodeJS publish/builds -->
+  <Target Name="_NodePrep" AfterTargets="_AfterWasmBuildApp" Condition="'$(ForNode)' == 'true'">
+    <!-- If package.json is in the directory copy it to new output directory -->
+    <Copy SourceFiles="package.json"
+        DestinationFolder="$(WasmAppDir)"
+        SkipUnchangedFiles="true" 
+        Condition="Exists('package.json')"/>
+
+    <!-- Otherwise create a new file in the output directory with the defaults -->
+     <WriteLinesToFile
+      File="$(WasmAppDir)\package.json"
+      Lines='{
+  "name": "mono-wasm-test",
+  "version": "1.0.0",
+  "main": "runtime.js",
+  "author": "Microsoft",
+  "license": "MIT",
+  "dependencies": {
+    "node-gyp-build": "^4.2.0",
+    "bufferutil": "^4.0.1",
+    "utf-8-validate": "^5.0.2",
+    "ws": "^7.5.0"
```

Comment:
Just to clarify these dependencies, we use the `ws` (websockets) one. The rest are what NPM calls peer-dependencies. However, since CI is using an older version of NPM (6.x.x) it does not install these dependencies automatically. So I set them explicitly.